app/helpers/application_helper.rb

```diff
@@ -179,6 +179,6 @@ def manifest_url(resource)
   # Generate the path for the Universal Viewer iframe @src attribute
   # @return [String]
   def universal_viewer_path(resource)
-    "/uv/uv#?manifest=#{manifest_url(resource)}"
+    "/uv/uv#?manifest=#{manifest_url(resource)}&config=#{root_url}/uv/uv_config.json"
```

Comment:
In `/viewer.html` we set the config explicitly:
https://github.com/pulibrary/figgy/blob/master/public/viewer.html#L12

But in `/uv/uv.html` we expect this param:
https://github.com/pulibrary/figgy/blob/master/public/uv/uv.html#L46

Can we just set the config explicitly in both places to make sure they are consistent, and always using our config?  Or is there some reason why we might want to override the config in some cases but not others?

Reply:
If we want to change it we should switch to viewer.html, the other one is that way because it's how the file that ships with UV works, so it gets overridden when we update.

Reply:
I think that would be good — that's what we're using from other apps, right?  Being consistent on which viewer HTML file we're using seems like it'll prevent bugs where the viewer works right some places but not others.

Reply:
No, we use UV.html in the other apps. Shaun said he had some css problems with viewer.html.